manual/embedding.rst

```diff
@@ -122,7 +124,8 @@ Several Julia values can be pushed at once using the ``JL_GC_PUSH2`` , ``JL_GC_P
 控制垃圾回收
 ---------------------------------------------------
 
-There are some functions to control the GC. In normal use cases, these should not be necessary.
+..There are some functions to control the GC. In normal use cases, these should not be necessary.:
+有一些函数可以帮助控制垃圾回收。但在一般情况下，它们都不需被专门使用。
```

Comment:
Addressed as you suggested.

Next time I'll translate more contents for one pull request.